.changeset/breezy-boxes-press.md

```diff
@@ -0,0 +1,19 @@
+---
+'xstate': major
+---
+
+Strict mode is no longer available. Instead, wildcard transitions should be used to explicitly handle undesired events:
```

Comment:
This sort of has already been removed from the `next` branch but it might not be obvious to users and since this changeset mentions "strict mode" being removed I think it's worth mentioning that here too. In v4 the `.strict` property has also controlled the behavior related to rejected promise actors:
https://github.com/statelyai/xstate/blob/80bfe1542f1d10261698562c80954aa8bad81246/packages/core/src/interpreter.ts#L1058

Reply:
> This sort of has already been removed from the `next` branch but it might not be obvious to users and since this changeset mentions "strict mode" being removed I think it's worth mentioning that here too.

Sorry, what should be mentioned exactly in the changeset?